articles/create-a-project.mdx

```diff
@@ -0,0 +1,84 @@
+---
+_schema: default
+_uuid: b73fc0b8-1c78-48d9-b7e0-2ecd6e7a1dd1
+_created_at: '2021-09-15 23:13:40 +0000'
+title: Create a Project
+nav_title: Create a Project
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/projects/2023-Q4/CloudCannon-Documentation-Project-Sites-Page.png
+description: >-
+  Learn how to group multiple sites from the same repository into a Project with
+  CloudCannon.
+tags:
+  - Projects
+  - Git
+  - Branching
+related_articles:
+  - articles/add-related-links-to-your-project.mdx
+related_links:
+---
+<comp.Notice info_type="important">
+  Some features of the advanced publishing workflow are only available on our <a rel="noopener noreferrer" target="_blank" href="https://cloudcannon.com/pricing/">Team and Enterprise plans</a>.
+</comp.Notice>
+
+Projects allow you to group multiple sites from the same repository. You can set up a main branch and enable team members to create branch sites and merge them back to the main branch at a later date.
+
+## Adding a new Project
+
+To create a project:
+
+1. Click the *Projects* section on the CloudCannon sidebar.
+2. Click the *\+ Create Project* button in the top right of the *Projects* page.
+3. Select your Git Provider and Repository. This can be a repository you have already connected to a site or an unconnected repository.
+4. Select the name of your Main Branch. This needs to match the name of the branch in your Git Provider.
+5. Click the *Create Project* button. If you have already connected this repository to a site, CloudCannon will automatically add that site to your new Project.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/projects/2023-Q4/CloudCannon-Documentation-Create-First-Project.png" alt="A screenshot of the CloudCannon app shows the Projects page with a 'create your first project' button." type="screenshot"/>
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/projects/2023-Q4/CloudCannon-Documentation-Create-Project.png" alt="A screenshot of the CloudCannon app shows a field for your git provider, repository, and main branch name." type="screenshot"/>
+
+## Set your main branch
+
+You now have a project! Let’s add your first site - which is usually your main branch.
```

Comment:
"... first site, which is..."
or 
"... first site — this is usually your..." (note also em-dash here, not hyphen)